tests/foreman/ui/test_repository.py

```diff
@@ -1615,6 +1615,43 @@ def test_positive_upload_rpm_non_admin(self):
             self.assertIn(RPM_TO_UPLOAD.rstrip('.rpm'), packages)
 
     @tier1
+    def test_positive_create_non_admin(self):
+        """Create yum repository via UI by non-admin user
+
+        :id: 6af5357e-d200-49e0-bf41-6d977b732810
+
+        :expectedresults: repository is successfully created
+
+        :BZ: 1398574, 1437134
+
+        :CaseImportance: Critical
+        """
+        role = entities.Role().create()
+        entities.Filter(
+            permission=entities.Permission(
+                resource_type='Katello::Product').search(),
```

Comment:
Strange that name is still `Katello::Product` when in UI we have `product and repositories` resource type